### ISSUE TITLE
PF.conf does not include ntp

### ISSUE THREAD

**Issue by tonyhain** [fusionpbx/fusionpbx#2675]
The system should have accurate time for CDRs, and blocking ntp through the firewall doesn't help that. Add to pf.conf
 
pass in quick inet proto udp from any to any port 123 keep state
pass in quick inet proto tcp from any to any port 123 keep state
pass in quick inet6 proto udp from any to any port 123 keep state
pass in quick inet6 proto tcp from any to any port 123 keep state

**Comment by mafoo** [comment]
Fusion's install system doesn't currently  configure ntp so it wouldn't arbitrary open it's ports. from the fact you are referring to PF i take it you are running freebsd?

**Comment by markjcrane** [comment]
I could be wrong but the client should connect from the server that has access to any port going out of the server. When run from the inside it would have permission for NTP. With that would it really be necessary to keep the port open inbound for the NTP ports?

**Comment by tonyhain** [comment]
Mafoo, you are correct, but given that pf wasn't installed prior to adding fusion, it broke remote access to that ntp server.

Mark, it appears on the systems I did a quick test on that the ntp client does work with the provided configuration. I tripped over this because pf was not previously installed on the target server, so the ntp server port was blocked by installing fusionpbx. 

It is not a big deal, but if the install script could do 
netstat -an | grep ^udp | awk '{print $4}' | grep 123$
then add the previous lines if that is not blank, you would not break an existing ntp server. That is not a portable cross-platform approach, so it may not be worth the effort.
